packages/vscode-extension/PRERELEASE.md

```diff
@@ -2,6 +2,82 @@
 
 ## Changelog
 
+### Apr 18, 2023
+
+New features:
+
+- Run and debug Teams bot and message extension applications with Dev Tunnel.
```

Comment:
"Run and debug Teams bot and message extension applications with dev tunnels in Visual Studio Code."

How do devs switch to use ngrok? Can we add a note that it's still possible to use ngrok?